ansible/README.md

```diff
@@ -1,10 +1,42 @@
 # Ansible Playbooks
 
+## Database Initialization
+
+The following databases are created by the `postgresql/init.yml` playbook:
+
+| Database      | Owner   |
+| ------------- | ------- |
+| de            | de      |
+| notifications | de      |
+| metadata      | de      |
+| de_releases   | de      |
+| grouper       | grouper |
+| qms           | de      |
+| unleash       | de      |
+| k3s           | de      |
+
+The owner users are configurable through the `dbms_connection_user` and `grouper_connection_user` group_vars.
+
+Migrations are run for the `de`, `metadata`, `notifications`, and `de_releases` databases. The `k3s` database is initialized by the installation process for the k3s cluster and the `grouper` database is handled by it's own playbook since it's fairly complicated. The `qms`
```

Comment:
There's a typo on this line.

```suggestion 
Migrations are run for the `de`, `metadata`, `notifications`, and `de_releases` databases. The `k3s` database is initialized by the installation process for the k3s cluster and the `grouper` database is handled by its own playbook since it's fairly complicated. The `qms`
```